randovania/gui/lib/common_qt_lib.py

```diff
@@ -177,6 +177,16 @@ def prompt_user_for_preset_file(window: QtWidgets.QWidget, new_file: bool, name:
     )
 
 
+def prompt_user_for_preset_folder(window: QtWidgets.QWidget) -> None | Path:
```

Comment:
this function being in common_qt_lib is weird, is there a specific reason why it needs to be here?
Considering its only used once, can you not just inline it?

Reply:
That's where all the other ones were. The _prompt_user_for_directory function is prefixed with an underscore suggesting it's not supposed to be consumed as a public API, so I followed the pattern of the rest of the file.

Reply:
it is not tho?
![grafik](https://github.com/randovania/randovania/assets/38186597/f6f91456-f73c-48de-8e74-1a0a1e5f10f9)


I dont see any leading underscore.